Deleted weave logic for default sku

MarekB01 opened the pull request:
I have deleted weave logic for using default SKU in inventory for version 6.3

https://github.com/BroadleafCommerce/QA/issues/4686

MarekB01 committed:
BroadleafCommerce/QA-4674_version_6_3

I have deleted weave logic for using default SKU in inventory for version 6.3

riteshadhikari17 commented on core/broadleaf-framework/src/main/resources/bl-framework-applicationContext.xml:
```diff
@@ -257,4 +257,39 @@
         <property name="collectionRef" value="blMergedCacheConfigLocations-framework"/>
         <property name="targetRef" value="blMergedCacheConfigLocations"/>
     </bean>
+
```
I think you need to remove these. There is error `Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No bean named 'blProductInventoryUseDefaultSku' available`.

MarekB01 committed:
BroadleafCommerce/QA-4674_version_6_3

I have deleted weave logic for using default SKU in inventory for version 6.3 . Also deleted bean which was related to old logic
